bookwyrm/templates/author/author.html

```diff
@@ -22,42 +22,59 @@ <h1 class="title">{{ author.name }}</h1>
     </div>
 </div>
 
-<div class="block content columns">
-    {% if author.aliases or author.born or author.died or author.wikipedia_link %}
+<div class="block content columns" itemscope itemtype="https://schema.org/Person">
+    <meta itemprop="name" content="{{ author.name }}">
+
+    {% if author.aliases or author.born or author.died or author.wikipedia_link or author.openlibrary_key or author.inventaire_id %}
     <div class="column is-narrow">
         <div class="box">
             <dl>
                 {% if author.aliases %}
                 <div class="is-flex">
                     <dt class="mr-1">{% trans "Aliases:" %}</dt>
-                    <dd itemprop="aliases">{{ author.aliases|join:', ' }}</dd>
+                    <dd>
+                        {% for alias in author.aliases %}
+                        <span itemprop="alternateName">{{alias}}</span>{% if not forloop.last %},{% endif %}
+                        {% endfor %}
+                    </dd>
                 </div>
                 {% endif %}
                 {% if author.born %}
                 <div class="is-flex">
                     <dt class="mr-1">{% trans "Born:" %}</dt>
-                    <dd itemprop="aliases">{{ author.born|naturalday }}</dd>
+                    <dd itemprop="birthDate">{{ author.born|naturalday }}</dd>
                 </div>
                 {% endif %}
-                {% if author.aliases %}
+                {% if author.died %}
```

Comment:
I stumbled on a bug I didn’t fill about a `None` value displaying the absence of a date of death. We can either fix that later or now at your own convenience. :)

(example: https://bookwyrm.social/author/18686)

Reply:
Already done in #1130 (I just have to update this branch with `main`)